### DISCUSSION TITLE
Clicking on a URL icon doesn't open url, expands node

### DISCUSSION THREAD

**Discussion by rouilj** [freeplane/freeplane#1625] [Help] [answered]
Hi all.

I have been using freeplane for more years than I can remember. It has been a great tool.
I am running 1.11.8. If I:

  1. open a map, create a node and add a text body (it happens w/o the text body too).
  2. hit ^K and add an HTTP/HTTPS (not on local disk) URL.
  3. I get an icon for the url.
  4. If I hover over the icon, I can see the url displayed in the status area.
  5. Now add two subnodes to the newly created node.
  6. If the node is not already selected (a light blue background is displayed), clicking on the url icon
      selects the node.
  7. With the newly added subnodes, if the first node is selected, clicking expands/collapses the node children.
      Without the child nodes nothing happens.

It used to be that clicking on the url icon would open the web page in my default browser.
I am not sure when it changed.

Is anybody else seeing this? If not is there some setting I may have inherited or set that causes this?
Is there some way to get a menu/click open to open the url?

OS windows 10 version 1.11.8. Installed using the embedded java installer.
About reports (btw it would be nice to be able to copy all this info easily, each line has to be selected individually now.)
```
297aad4e0654f3a02747f1cd8ebd65de97306889 dpolivaev 2023-11-12T20:46:51Z
Java version: 15.0.10
Installation resources: C:\Program Files\Freeplane\resources
```

I do have some plugins installed:
```
  jumper 1.0.1
  MapInsight 1.2.0
  Markdown Helper 0.0.0
  NodeMagnet 0.0.2
  GTD sync 0.7.7
  FreePlane GTD 0.8.1
  Free IBIS 0.1.1
  Bookmarks 0.7.2
```

Also I am getting a security error that says I have disabled plugins/scripts. It directs me to tools/preferences/plugins, but I don't have anything in that frame that would enable/disable scripts. The errors aren't generated in response to my clicking on the url icon. It may be a problem but I don't think it is related to this problem.

Thanks for any ideas.

**Comment by i-plasm** [comment]
Hi rouilj, I suggest you deactivate all your plugins and test if the problem persists. Steps:
1. `Tools > Add-ons `
1. Click `Deactivate` for all plugins
1. Restart freeplane
1. Check if the problem persists

**Reply by rouilj** [reply]
Hi @i-plasm:

All the plugins are deactivated and freeplane restarted. No change.

(Note it would be nice to have a 'deactivate/activate all plugins' option/button on the add-on's page.)

Thanks for the suggestion. Any other ideas? Can you replicate it in one of your maps?

**Comment by dpolivaev** [comment, marked as answer]
Check if option "Preferences…->Behaviour->Open node link on simple mouse click"  is enabled.

**Reply by rouilj** [reply]
Thanks, "Preferences…->Behaviour->Open node link on simple mouse click" was not enabled. I wonder how it got turned off ... In any case thanks again.
